docs/application/native/guides/applications/overview.md

```diff
@@ -18,7 +18,7 @@ Tizen provides various application models to allow you to create applications ta
 
 - [Component-based Applications](component-based-app.md)
```

Comment:
@gotoextreme 
As we have changed "Component-based" to "Component based".

Component Based Applications

Reply:
Done